hawk/app/models/template.rb

```diff
@@ -194,165 +180,8 @@ def available_meta(params = {})
-    def available_ops(params = {})
-      path = [
-        params[:clazz],
-        params[:provider],
-        params[:type]
-      ].compact.reject(&:empty?).join(":")
-
-      xml = REXML::Document.new(
-        Util.safe_x(
-          "/usr/sbin/crm_resource",
-          "--show-metadata",
-          path
-        )
-      )
-
-      return {} unless xml.root
-
-      {}.tap do |result|
-        xml.elements.each("//action") do |el|
-          name = el.attributes["name"]
-
-          available = Hash[el.attributes.collect { |a| a.to_a }]
-          available.delete "name"
-          available.delete "depth"
-
-          ops = {}.tap do |result|
-            available.each do |key, default|
-              result[key] = {
-                type: "string",
-                default: default
-              }
-            end
-          end
-
-          if name == "monitor"
-            unless ops.has_key?("interval")
-              key = "interval"
-              ops[key] ||= {}
-              ops[key][:default] = "20"
-              ops[key][:required] = true
-            end
-
-            key = "OCF_CHECK_LEVEL"
-            ops[key] ||= {}
-            ops[key][:default] = "0"
-            ops[key][:type] = "string"
-          end
-
-          result[name] ||= {
-            "interval" => {
-              type: "string",
-              default: 0,
-              required: false
-            },
-            "timeout" => {
-              type: "string",
-              default: "0",
-              required: true
-            },
-            "requires" => {
-              type: "enum",
-              default: "fencing",
-              values: [
-                "nothing",
-                "quorum",
-                "fencing"
-              ]
-            },
-            "enabled" => {
-              type: "boolean",
-              default: "true"
-            },
-            "role" => {
-              type: "enum",
-              default: "",
-              values: [
-                "Stopped",
-                "Started",
-                "Slave",
-                "Master"
-              ]
-            },
-            "on-fail" => {
-              type: "enum",
-              default: "stop",
-              values: [
-                "ignore",
-                "block",
-                "stop",
-                "restart",
-                "standby",
-                "fence"
-              ]
-            },
-            "start-delay" => {
-              type: "string",
-              default: "0"
-            },
-            "interval-origin" => {
-              type: "string",
-              default: "0"
-            },
-            "record-pending" => {
-              type: "boolean",
-              default: "false"
-            },
-            "description" => {
-              type: "string",
-              default: ""
-            }
-          }
-
-          result[name].merge! ops
-        end
-      end
+    def available_opmeta(params = {})
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used. You can also write as `available_opmeta(*)` if you want the method to accept any arguments but don't care about them.